### DISCUSSION TITLE
Annotation on Role, RoleBinding

### DISCUSSION THREAD

**Discussion by mglants** [strimzi/strimzi-kafka-operator#7832] [Q&A] [answered]
Hello, I'am using HNC https://github.com/kubernetes-sigs/hierarchical-namespaces
It's trying to propagate entity-operator roles and rolebinding to children namespaces, so strimzi alerting it's can't modife rolebinding in chidlren namespace as HNC blocking it.
So anyway to add annotations to Role,Rolebindings creating when deploying kafka with entity-operator

**Comment by scholzj** [comment, marked as answer]
I do not think there is any way to configure the metadata for EO Role and RoleBinding. This would be something what could be possibly added to the `template` section (`.spec.entityOperator.template` - https://strimzi.io/docs/operators/latest/full/configuring.html#type-EntityOperatorTemplate-reference).

If you are interested in contributing it, we are definitely opened to PRs for this. If not, you can still open an enhancement issue and someone else might / will get to it: https://github.com/strimzi/strimzi-kafka-operator/issues
